src/interfaces/IdToken.tsx

```diff
@@ -0,0 +1,3 @@
+export interface IIdToken {
+  idToken: string
+}
```

Comment:
Muda isso para `IToken` e `token`.

Ai, quando você for passar no body da API, você vai mandar assim:
```
    await api.post('user/login', {
        token,
      })
```